examples/howto/events_app.py

```diff
@@ -27,13 +27,19 @@ def display_event(div, attributes=[]):
     l2 = "div.text = '<b>JS Events (see console for Python events)</b><br><font size=\"0.5pt\">' + text.split('<br>',20).join('<br>')+ '</font>';"
     return CustomJS(code=l1+l2, args={'div': div})
 
+def pprint(cls, attributes):
+    ''' Pretty print the event with the given set of attributes'''
+
 
 def print_event(attributes=[]):
     """
     Function that returns a Python callback to pretty print the events.
     """
     def python_callback(event):
-        print(event.pprint(attributes))
+        cls_name = event.__class__.__name__
+        attrs = ', '.join(['{attr}={val}'.format(attr=attr, val=event.__dict__[attr])
+                       for attr in attributes])
+        print('{cls_name}({attrs})'.format(cls_name=cls_name, attrs=attrs))
```

Comment:
One alternative might be to have a ``repr`` for events that includes the ``model_id`` and a ``str`` that doesn't. The idea them is that the ``repr`` has everything needed to re-create the event from the representation whereas the ``str`` is more readable (which means it could be used here). Does that seem reasonable?